app/code/Magento/ReleaseNotification/Ui/DataProvider/NotificationDataProvider.php

```diff
@@ -133,82 +133,76 @@ public function setConfigData($config)
     }
 
     /**
-     * {@inheritdoc}
-     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     * @inheritdoc
      */
     public function getFieldMetaInfo($fieldSetName, $fieldName)
     {
         return [];
     }
 
     /**
-     * {@inheritdoc}
-     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
```

Comment:
Removing this annotation will cause static test failures because parameters are not used in the method.